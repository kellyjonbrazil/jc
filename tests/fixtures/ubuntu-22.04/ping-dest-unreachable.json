{"destination_ip":"10.0.3.22","data_bytes":56,"pattern":null,"destination":"10.0.3.22","duplicates":0,"packets_transmitted":3,"packets_received":0,"errors":3,"packet_loss_percent":100.0,"time_ms":2049.0,"responses":[]}
