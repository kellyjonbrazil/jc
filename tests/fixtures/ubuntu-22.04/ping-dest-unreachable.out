PING 10.0.3.22 (10.0.3.22) 56(84) bytes of data.
From 10.0.0.1 icmp_seq=1 Destination Host Unreachable
From 10.0.0.1 icmp_seq=2 Destination Host Unreachable
From 10.0.0.1 icmp_seq=3 Destination Host Unreachable

--- 10.0.3.22 ping statistics ---
3 packets transmitted, 0 received, +3 errors, 100% packet loss, time 2049ms
pipe 3
